src/test/resources/features/patch.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Update a resource with PATCH request 
+
+Background:
+Given a file with the name will-be-patched.json exists in /
+
+Scenario: Unsupported PATCH document
+  When a client makes a PATCH request to /will-be-patched.json
+  And the request contains the text/plain message body "plain text patch"
+  Then the server should respond with status code 415 Unsupported Media Type
+  And the server should respond with the header Accept-Patch application/json-patch+json
+
+Scenario: Resource not found 
+  Given a file with the name does-not-exist.json does not exist in /
+  When a client makes a PATCH request to /does-not-exist.json
+  And the request contains the application/json message body 
+  Then the server should respond with status code 404 Not Found
```

Comment:
I forget - were you unable to test successful patch requests with cucumber?

Reply:
I wasn't able to get the the Cucumber step definition to write default text to does-not-exist.json. The test did work if I manually created the file and populated the content, but then the test would only work once. Depending on the JSON patch operation that I tested, it would continue to show the test as passing (such as when adding a new key/value pair), but it would fail after the first time if I deleted a key/value pair. I ended up time-boxing the last acceptance test because it was taking up so much time